frontend/src/pages/profile/components/avatar-wrapper/avatar-wrapper.tsx

```diff
@@ -0,0 +1,110 @@
+import { type ChangeEvent, type FC } from 'react';
+
+import { Avatar, ErrorMessage, Icon } from '~/libs/components/components.js';
+import { IconButton } from '~/libs/components/icon-button/icon-button.js';
```

Comment:
`~/libs/components/components.js `

Reply:
Thanks for having my back

- [65654a1](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/118/commits/65654a1286d0d5908f927dd8b03a652952aea9d5)